src/hardwareModel.py

```diff
@@ -457,3 +464,22 @@ def gen_cacti_results(self):
             else 0.0
         )
         return
+
+    def get_wire_parasitics(self, arg_parasitics, arg_testfile):
+        os.system("cp " + arg_testfile + " ./" + directory) 
+        os.system("cp openroad_interface/tcl/codesign_flow.tcl ./" + directory) 
+        shutil.copyfile(arg_testfile, directory + "test.tcl")
+        design_name= self.path_to_graphml.split("/")[len(self.path_to_graphml.split("/")) - 1]
+        graph = nx.read_gml(self.path_to_graphml)
+        graph, net_out_dict, node_output, lef_data, node_to_num= def_generator.def_generator(arg_testfile, graph)
+
+        if arg_parasitics == "detailed":
+            _, graph = place_n_route.detailed_place_n_route(graph, design_name, net_out_dict, node_output, lef_data, node_to_num)
+        elif arg_parasitics == "estimation":
+            _, graph = place_n_route.estimated_place_n_route(graph, design_name, net_out_dict, node_output, lef_data, node_to_num)
+        elif arg_parasitics == "none":
+            graph = place_n_route.none_place_n_route(graph, design_name, net_out_dict, node_output, lef_data, node_to_num)
+    
+        self.parasitic_graph = graph
```

Comment:
didn't we create a function in place_n_route that does exaclty this? We should be calling that function here.